src/jailer/src/env.rs

```diff
@@ -827,6 +833,15 @@ mod tests {
         args.parse(&make_args(&invalid_parent_cg_vals)).unwrap();
         assert!(Env::new(&args, 0, 0).is_err());
 
+        let invalid_controller_pt = ArgVals {
+            cgroups: vec!["./root/", "../root"],
```

Comment:
It's maybe best if we have 2 tests here: one for when the path contains ./ or ../ and another for when the cgroup is specified in the wrong format (not in cgroup_file=value format). With this test we can get an error either if the parameter contains current/parent dir or if it does not have the proper format.